### PR TITLE
 added the new twitter logo(x)

### PR DESCRIPTION
**Pull request by akshaykaluse35**
This pull request introduces the updated Twitter logo(x) to the project. The new logo(x) enhances brand representation and aligns with the latest design guidelines provided by Twitter. The changes include updating the logo(x) asset files and integrating them into the project's codebase.

Changes:

    Added the new Twitter logo(x) asset files.
    Updated references to the Twitter logo(x) in relevant project files.
    Ensured consistency with the project's design standards.

Reasoning:
Keeping the project up-to-date with the latest branding assets is essential for maintaining a cohesive user experience. By incorporating the new Twitter logo(x), we ensure that our application remains visually appealing and in line with current design trends.

Testing:

    Verified that the new Twitter logo(x) displays correctly across various screen resolutions.
    Checked for any rendering issues or distortions caused by the logo(x) update.
    Tested logo(x) accessibility and visibility on different backgrounds and themes.

### CODE REVIEW

**Comment by shubham-kshetre** [comment]
Hello @akshaykaluse35, The preview of your PR is working fine. But, icon changed by you seems to be blank to me. PTAL!

**Comment by shubham-kshetre** [comment]
Also, Please do changes in your forked repository instead of organization repository

**Comment by akshaykaluse35** [comment]
@shubham-kshetre I've tested it and it works fine for my end

**Comment by akshaykaluse35** [comment]
@shubham-kshetre yeah sure ;) next time i'll make changes to forked repo

**Comment by shubham-kshetre** [comment]
@akshaykaluse35 can you PTAL for why icon is not visible in the commit?

**Comment by shubham-kshetre** [comment]
@SAYUK09 PTAL

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@fontsource/inter": "4.5.2",
     "@giscus/react": "^2.4.0",
     "@mailchimp/mailchimp_marketing": "^3.0.58",
-    "@next/bundle-analyzer": "^12.1.4",
+    "@next/bundle-analyzer": "^12.3.4",
```
Comment (SAYUK09) [inline package.json:18]: Why is there a change in package.json file?
Did you install any packages ?

**Review by SAYUK09 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (akshaykaluse35) [reply package.json:18]: I haven't installed any packages.... Thank You ;)

**Review by shubham-kshetre — APPROVED** [review]

**Review by SAYUK09 — APPROVED** [review]

**Pull request merged**